Mathlib/Algebra/Parity.lean

```diff
@@ -273,24 +276,23 @@ alias ⟨Even.exists_bit0, _⟩ := even_iff_exists_bit0
 
 section Semiring
 
-variable [Semiring α] [Semiring β] {m n : α}
+variable [Semiring α] [Semiring β] {a b : α} {m n : ℕ}
```

Comment:
Can you do this rename in a standalone PR with F2?

Reply:
#12260 but I think this is split-abuse. This PR is already split off #11855 and #11863